db/migrate/20220823083657_change_case_log_to_lettings_log.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeCaseLogToLettingsLog < ActiveRecord::Migration[7.0]
+  def change
+    rename_table :case_logs, :lettings_logs
```

Comment:
I didn't do this because it wouldn't be a zero downtime deploy. The old application instance would error as it would still be reading from the previous table.

Reply:
True but the rename should be pretty close to instantaneous so given the usage volume I wouldn't expect it to be much of an issue.

Reply:
The rename will be quick, but the upper bound is the time it takes to switch the instances. Your call!